SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -339,6 +339,7 @@ extern const FString SpatialSessionIdURLOption;
 extern const FString LOCATOR_HOST;
 extern const FString LOCATOR_HOST_CN;
 const uint16 LOCATOR_PORT = 443;
```

Comment:
Would we want to change this one to `uint16_t` too?

Reply:
We definitely want, thanks for pointing it out :)